### PR TITLE
fix for og image undefined description issue.

### PR DESCRIPTION
**Pull request by R041T**
## What kind of change does this PR introduce?

Bug fix for _og image description undefined #14229_

## What is the current behavior?

When a Docs og images has a description that is undefined the og image generator displays the undefined description.

## What is the new behavior?

It now doesn't render the description if its undefined.

Screenshot 2023-05-09 at 11 17 36 PM

### CODE REVIEW

**Comment by R041T** [comment]
Hi, this is my first open source contribution. Please let me know if there is any change I need to make. Thanks!

File: supabase/functions/og-images/handler.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export async function handler(req: Request) {
 
   switch (site) {
     case 'docs':
-      return new ImageResponse(( <Docs title={title} description={description} type={type} icon={icon} /> ),
+      return new ImageResponse(( <Docs title={title} description={(description !=='undefined')? description:''} type={type} icon={icon} /> ),
```
Comment (asjadaugust) [inline supabase/functions/og-images/handler.tsx:33]:
```suggestion
      return new ImageResponse(( <Docs title={title} description={(description || ''} type={type} icon={icon} /> ),
```

Comment (R041T) [reply supabase/functions/og-images/handler.tsx:33]: Hi, issue persists with the suggested change.

Screenshot 2023-05-10 at 7 52 13 AM

Comment (asjadaugust) [reply supabase/functions/og-images/handler.tsx:33]: Ah, never mind, I assumed description to be undefined and not the string 'undefined'

**Review by Isaiah-Hamilton — APPROVED** [review]
lgtm!

File: supabase/functions/og-images/handler.tsx (hunk 1 above)
Comment (gregnr) [reply supabase/functions/og-images/handler.tsx:33]: Curious if you know where the string `'undefined'` is coming from? Is there a place up the chain we should be addressing the fix?

Comment (R041T) [reply supabase/functions/og-images/handler.tsx:33]: Hi, It's coming in directly as a description query parameter. I'm not exactly sure why the description 'undefined' needs to be handled specifically. Perhaps @Isaiah-Hamilton can provide the context.

Comment (gregnr) [inline supabase/functions/og-images/handler.tsx:33]: Just some formatting to be consistent:
```suggestion
      return new ImageResponse(( <Docs title={title} description={description !== 'undefined' ? description : ''} type={type} icon={icon} /> ),
```

Comment (gregnr) [reply supabase/functions/og-images/handler.tsx:33]: Got it - must be on the Next.js SSG/SSR side. We can merge this as is and look into that at a later time.

**Pull request merged**